build.gradle

```
@@ -194,7 +194,7 @@ subprojects {
                 maven(MavenPublication) {
                     from project.components.java
 
-                    groupId = (project.name == 'apktool-lib' || project.name == 'brut.j.util') ? 'app.revanced' : 'org.apktool'
+                    groupId = 'app.revanced'
```

Publishing packages for modules of group `app.revanced` should fix the issue:

```suggestion
                    groupId = project.name != 'brut.j.util' ? 'app.revanced' : 'org.apktool'
```